src/Gui/Application.cpp

```diff
@@ -2112,6 +2113,8 @@ void Application::initOpenInventor()
     SoDB::init();
     SIM::Coin3D::Quarter::Quarter::init();
     SoFCDB::init();
+    SoFCDB::init();
+    SoDevicePixelRatioElement::initClass();
```

Comment:
Move this line into SoFCDB::init where we initialize all our custom Inventor classes.

Reply:
Done. Please check again and merge if this is ok now.
Thanks